package-lock.json

```diff
@@ -31574,7 +31576,7 @@
         "@types/react": "^17.0.37",
         "@types/react-dom": "^17.0.11",
         "ansi-to-react": "^6.1.6",
-        "buffer": "*",
+        "buffer": "^6.0.3",
```

Comment:
Thanks.

Reply:
🤔 Hah, this just happened on it's own I guess?